test/benchmark.rb

```diff
@@ -1,87 +1,28 @@
 # frozen_string_literal: true
 
 require "benchmark/ips"
-require "html/pipeline"
-require "commonmarker"
 require "sanitize"
 require "selma"
-require_relative "benchmark/selma_config"
-
-REWRITE_INPUT = File.read("test/benchmark/rewrite_benchmark_input.md").freeze
-
-def bytes_to_megabytes(bytes)
-  (bytes.to_f / 1_000_000).round(2)
-end
 
 DIR = File.expand_path(File.dirname(__FILE__))
 
 DOCUMENT_HUGE   = File.read("#{DIR}/benchmark/html/document-huge.html").encode("UTF-8", invalid: :replace, undef: :replace)
 DOCUMENT_MEDIUM = File.read("#{DIR}/benchmark/html/document-medium.html").encode("UTF-8", invalid: :replace, undef: :replace)
 DOCUMENT_SMALL  = File.read("#{DIR}/benchmark/html/document-small.html").encode("UTF-8", invalid: :replace, undef: :replace)
 
-FRAGMENT_LARGE = File.read("#{DIR}/benchmark/html/fragment-large.html").encode("UTF-8", invalid: :replace, undef: :replace)
-FRAGMENT_SMALL = File.read("#{DIR}/benchmark/html/fragment-small.html").encode("UTF-8", invalid: :replace, undef: :replace)
-
 def compare_sanitize
   sanitize_config = Sanitize::Config::RELAXED
-  [[DOCUMENT_HUGE, "huge"], [DOCUMENT_MEDIUM, "medium"], [DOCUMENT_SMALL, "small"]].each do |(html, label)|
     Benchmark.ips do |x|
-      x.report("sanitize-document-#{label}") do
```

Comment:
Shortened the output so that it no longer breaks the line:

**original**

```
Calculating -------------------------------------
sanitize-document-small
                        159.469 (± 0.6%) i/s -    798.000 in   5.004561s
selma-document-small      1.832k (± 1.7%) i/s -      9.180k in   5.012809s
```

**revised**

```
Calculating -------------------------------------
      sanitize-small    140.647 (± 2.8%) i/s -    715.000 in   5.087293s
         selma-small      1.924k (± 1.5%) i/s -      9.666k in   5.024335s
```